app/discord.server.ts

```diff
@@ -0,0 +1,46 @@
+import crypto from "node:crypto";
+import DiscordOauth2 from "discord-oauth2";
+
+const DISCORD_REDIRECT_URI =
+  process.env.NODE_ENV === "production"
+    ? "https://opensourcehub.io/verify-discord"
+    : "http://localhost:3000/verify-discord";
+
+const DISCORD_SCOPES = ["identify"];
```

Comment:
**Suggestion (non-blocking)**: Is there any documentation of Discord's available scopes? Maybe add a link to such documentation in a comment?